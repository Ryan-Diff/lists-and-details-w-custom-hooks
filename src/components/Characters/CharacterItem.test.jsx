import React from 'react';
import { render } from '@testing-library/react';
import Character from './CharacterItem';

describe('Character component', () => {
    it('renders a character', () => {
        const { asFragment } = render(
            <Character
              name='Ryan'
              photoUrl='Test'
            />);
            expect(asFragment()).toMatchSnapshot();
    });
});
