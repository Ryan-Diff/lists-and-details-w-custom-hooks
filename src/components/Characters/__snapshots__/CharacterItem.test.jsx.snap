// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders a character 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="Ryan"
    />
    <figcaption>
      Ryan
    </figcaption>
  </figure>
</DocumentFragment>
`;
